src/include/miopen/find_controls.hpp

```diff
@@ -85,8 +85,9 @@ class MIOPEN_INTERNALS_EXPORT FindEnforce
     template <class Context>
     bool IsSearch(const Context& context) const
     {
-        return IsEnabled(context) &&
-               (action == FindEnforceAction::Search || action == FindEnforceAction::SearchDbUpdate);
+        return IsEnabled(context) && (action == FindEnforceAction::Search ||
+                                      action == FindEnforceAction::SearchDbUpdate ||
+                                      action == FindEnforceAction::DbUpdate);
```

Comment:
I added FindEnforceAction::DbUpdate here in Search too since[ this condition](https://github.com/ROCm/MIOpen/blob/develop/src/include/miopen/find_solution.hpp#L81) skips perf db loading.

Reply:
The perf db load skip condition flags on both IsSearch and IsDbUpdate. So there is no need to ambiguate these terms here.